emsdk.py

```diff
@@ -3047,13 +3047,12 @@ def print_tools(t):
       print('')
 
     tools_to_activate = currently_active_tools()
-    args = [x for x in args if not x.startswith('--')]
     for arg in args:
       tool = find_tool(arg)
       if tool is None:
         tool = find_sdk(arg)
-      if tool is None:
-        error_on_missing_tool(arg)
+        if tool is None:
+          error_on_missing_tool(arg)
```

Comment:
Yes, you will not see `no tool found: --arg` rather than silently ignoring it

Reply:
Right sorry, I guess what I meant was, should the error message be generalized to something like "unknown argument or no tool found by that name", since the error could now be either one of those cases.

Reply:
other command like `emsdk install` already report `error: tool or SDK not found: '--foo'`.. we could perhaps make that nicer, but that seems separate (and less important?) to ignoring args.